### DISCUSSION TITLE
群组的子节点隐藏,则子节点关联的边也隐藏

### DISCUSSION THREAD

**Discussion by SandyCheeks-wood** [antvis/X6#1700] [Q&A]
请教一下:
群组的子节点隐藏,则子节点关联的边也隐藏,有没有方法或者配置可以使边不隐藏?
或者修改边的target或source参数为隐藏子节点的父节点,
![image](https://user-images.githubusercontent.com/46735169/147639879-d4bc7c2a-b105-4ca7-b718-0d2a1d6e410d.png)

**Comment by NewByVector** [comment]
这个可以实现，实现逻辑：折叠父节点后，新增一条父节点之间的暂时的边。
